README.md

```diff
@@ -0,0 +1,174 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Todo List</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#Run-tests)
+- [🎥 Live Demo](#live-demo)
+- [🎥 Future Feature](#future-feature)
+- [👥 Authors](#authors)
+- [🎥 Project Presentation](#Project-presentation)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgments](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Todo List <a name="about-project"></a>
+
+> This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">css</a></li>
+  </ul>
+</details>
+<details>
+  <summary>js</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Use the low fidality design for the project😊👌
+- Use the semantics elements💯
+- Use the validation HTML for the form .🚀
+- Use contact form 😃👍
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A web browser to view output e.g [Google Chrome](https://www.google.com/chrome/).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/BilalLiaquat7/webpack.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd desktop`**
```

Comment:
- [ ] The cd command in the Readme should be directed to your repo name in this case `cd webpack`

<img width="269" alt="image" src="https://github.com/BilalLiaquat7/webpack/assets/41163571/6e60ddbb-44de-43d4-9fc6-38706c68e4d3">